tests/python/utils.py

```diff
@@ -406,7 +404,7 @@ def exec_nvfuser(
         *,
         new_fusion_expected=True,
         device=None,
-        is_clonable=False,
+        is_clonable=True,
```

Comment:
:clap:

Reply:
I reverted this. I still need to add `Matmul`, `SpdaFwd`, and `SpdaBwd`

Reply:
See https://github.com/NVIDIA/Fuser/pull/3240